renovate.json5

```diff
@@ -0,0 +1,29 @@
+{
+  $schema: 'https://docs.renovatebot.com/renovate-schema.json',
+  extends: ['local>netlify/renovate-config'],
+  // The config we're extending ignores test dirs, but we want to bump some fixture deps
+  ignorePaths: ['**/node_modules/**'],
```

Comment:
We'll probably want to update the `release-please` config as part of this PR

Reply:
I solved this with `extends: [':semanticCommitTypeAll(chore)']` instead!

Reply:
To be more specific: release-please doesn't trigger releases for `chore:` and Renovate uses a mix of `chore:` and `fix:` depending on the dependency. By telling it to only use `chore:` within the test fixture `package.json` bumps, we solve the problem of noisy no-op releases.

Reply:
The `chore:` here only applies to `remix`, but since we overwrite `ignorePaths` we now include package bumps inside fixture directories. This means that an upgrade to `@netlify/blobs` inside a fixture will trigger a release